testnets/xplatestnet/assetlist.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "xplatestnet",
+  "assets": [
+    {
+      "description": "The native staking token of XPLA.",
+      "denom_units": [
+        {
+          "denom": "axpla",
+          "exponent": 0
+        },
+        {
+          "denom": "xpla",
+          "exponent": 18
+        }
+      ],
+      "base": "axpla",
+      "name": "Xpla",
+      "display": "xpla",
+      "symbol": "XPLA",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/xpla/images/xpla.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/xpla/images/xpla.svg"
+      },
+      "coingecko_id": "xpla",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/xpla/images/xpla.png",
```

Comment:
this directory does not exist. Removing the /testnets/ part should fix

Reply:
Yes, thank you.
I've checked the wrong path and posted a commit that fixes it.